src/staging/src/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+
+export const sortByKey = (key) => {
```

Comment:
Nit: not seeing any reference to sortByKey. Could be removed, optional.

Reply:
Good catch! Removed the unused helper function.